quarkus/server/build.gradle.kts

```diff
@@ -51,6 +51,8 @@ dependencies {
     runtimeOnly(project(":polaris-eclipselink"))
   }
 
+  runtimeOnly(project(":polaris-relational-jdbc"))
```

Comment:
This dep looks premature. Le's add it when JDBC persistence is functional and can be integration-tested inside the Quarkus Server.